hera_cal/tests/test_red_groups.py

```diff
@@ -235,26 +236,26 @@ def test_keyed_on_bls(self):
             [[(0, 1)], [(0, 2), (0, 3)]],
         )
         print(rg._red_list)
-        new = rg.keyed_on_bls(bls=[(0,3)])
-        print(new._red_list)
-        assert (0,2) in new
-        assert new.get_ubl_key((0,2)) == (0,3)
-        assert new.get_ubl_key((0,1)) == (0,1)
-        print(new._red_list)
+        new = rg.keyed_on_bls(bls=[(0, 3)])
+        print(new._red_key_to_bls_map)
+        assert (0, 2) in new
+        assert new.get_ubl_key((0, 2)) == (0, 3)
+        assert new.get_ubl_key((0, 1)) == (0, 1)
 
         new.keyed_on_bls(bls=[(0, 2)], inplace=True)
-        print(new._red_list)
-        assert new.get_ubl_key((0,2)) == (0,2)
+        print(new._red_key_to_bls_map)
+        print(new._bl_to_red_map)
```

Comment:
Is there a reason we have all these print statements? Are they helpful?

Reply:
Nope. Cruft. Removed.